File: PHP/UberBlack.php
<?php

require_once('Car.php');

class UberBlack extends Car
{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function UberBlack($licence, $driver, $typeCarAccepted, $seatsMaterial,$passenger)
    {
        parent::__construct($licence, $driver,$passenger);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    
    typeCarAccepted = []
    seatsMaterial   = []
    
    def __init__(self,license,driver,typeCarAccepted,seatsMaterial)
        super.__init__(self,license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial   = seatsMaterial
        